src/oauth/helpers.test.ts

```diff
@@ -1,11 +1,21 @@
-import { describe, expect, it } from "vitest";
+import { eq } from "drizzle-orm";
+import { Hono } from "hono";
+import { beforeEach, describe, expect, it } from "vitest";
+import { cleanDatabase } from "../../tests/helpers";
+import {
+  createAccount,
+  createOAuthApplication,
```

Comment:
```suggestion
  createOAuthApplication,
  getAccessToken,
```

And then you can use that instead of doing the database query directly

Reply:
There's future work to rename that to `findAccessToken` though